### ISSUE TITLE
My "My Reports" text configurable

### ISSUE THREAD

**Issue by lindsayking** [Esri/crowdsource-reporter#270]
@allisonmuise  @ScottOppmann  @ChrisBuscaglia 

The "My Reports" text might not always be appropriate for all purposes.  For example, for Property Lister, the text might be better as "My Listings".  We would like to make this text configurable.

![image](https://cloud.githubusercontent.com/assets/4316336/22158169/d98d88cc-deef-11e6-802d-32a94b278f61.png)

**Comment by allisonmuise** [comment]
@lindsayking rather than making this configurable, how about making it more generic : 'My Submissions'?

**Comment by lindsayking** [comment]
@allisonmuise   I think it would be a nice touch to be able to make it configurable; it would allow organizations to make the app that much more specific for their purposes.

**Comment by allisonmuise** [comment]
@lindsayking @ScottOppmann @NikkiGolding @chris-fox 

Brainstormed about this with @ChrisBuscaglia yesterday and came up with a string that we think is a good balance between informative and use-case neutral. What do you guys think of 'My Contributions'? 

We liked it because it suits the collection of reports, ideas, comments, listings, etc and fits with the engagement side of the story - users are *contributing* to a larger initiative.

If you feel strongly against this, or have another idea, please comment! Otherwise, I'll be updating this string on Friday.

(REALLY don't want to add to the already massive list of configuration parameters, so hoping we can settle on a generic string)

**Comment by chris-fox** [comment]
If we are going to change it I would prefer "My Submissions" to "My Contributions". I don't think "My Contributions" works great for the problem reports.

**Comment by ScottOppmann** [comment]
Agree, don't think My Contributions works and would still like to pursue making this a configurable item.  Lets change it to "My Submissions" in near term and then add configurable item to backlog .

**Comment by allisonmuise** [comment]
string was updated for this release
